torch/csrc/jit/backends/backend.h

```diff
@@ -14,10 +15,11 @@ template <class TBackendInterface>
 class backend {
   static_assert(
       std::is_base_of<PyTorchBackendInterface, TBackendInterface>::value,
-      "torch::jit::backend_<T> requires T to inherit from PyTorchBackendInterface");
+      "torch::jit::backend<T> requires T to inherit from PyTorchBackendInterface");
   std::string backend_name_;
 
  public:
+  C10_DEPRECATED
```

Comment:
Oh cool, how does this work?

Reply:
Maybe we should mark preprocess on `PyTorchBackendInterface` as deprecated also?

Reply:
It's our in-house macro for the equivalent C++14 [[deprecated]] that handles portability across platforms.
Added it to PyTorchBackendInterface as suggested.